frontend/javascripts/oxalis/model/sagas/mapping_saga.ts

```diff
@@ -234,6 +234,12 @@ function* getLargestSegmentId(layerName: string): Saga<number> {
     throw new Error("Mappings only exist for segmentation layers.");
   }
 
+  if (segmentationLayer.largestSegmentId == null) {
+    // todo: fix this once the custom segment colors are merged into master
```

Comment:
Reminder to fix this ToDo.

Reply:
This is resolved now. Since the new custom segment color mechanism doesn't need reassignment of segment ids (the old code had to reassign the IDs so that the texture access was aligned with its 256**2 dimension), I don't even need to access the largest segment anymore. This is quite lucky, as otherwise I would have had to disable mappings with custom colors for segmentations with unknown largest segment id.
By the way, I also added an analytics event so that we know whether custom mapping colors are used at all.